vck-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oauth2/OAuth2ClientDPoPTest.kt

```diff
@@ -46,8 +46,8 @@ class OAuth2ClientDPoPTest : FunSpec({
                 issueRefreshTokens = true
             ),
         )
-        clientKey = EphemeralKeyWithSelfSignedCert()
-        jwsService = DefaultJwsService(DefaultCryptoService(clientKey))
+        clientKey = EphemeralKeyWithoutCert()
```

Comment:
do we have a behviouralc hange here (maybe combinde with other subtle changes)?

Reply:
There was no need to create a certificate for this test